lib/html_parser.dart

```diff
@@ -619,9 +621,27 @@ class HtmlRichTextParser extends StatelessWidget {
             if (node.attributes['src'] != null) {
               if (node.attributes['src'].startsWith("data:image") &&
                   node.attributes['src'].contains("base64,")) {
+                precacheImage(
+                  MemoryImage(
+                    base64.decode(
+                      node.attributes['src'].split("base64,")[1].trim(),
+                    ),
+                  ),
+                  buildContext,
+                  onError: (_, stacktrace) {
+                    print(stacktrace);
```

Comment:
Rather than just printing the error, do you think it would be helpful to have an `onError` (or perhaps `onImageError`) callback on the `Html` widget itself so that we can choose what to do when an error occurs?

Reply:
Yes I also thought about it but in my opinion there was no use case. What do you mean ?

Reply:
Hey @Sub6Resources what do you think ? Should I add a callback ?

Reply:
@The-Redhat Yeah, I think that would be best. By default the callback shouldn't do anything.

Reply:
Ok

Reply:
Ok